src/main/java/org/folio/circulation/domain/CreateRequestService.java

```diff
@@ -92,11 +94,11 @@ private static RequestAndRelatedRecords setRequestQueuePosition(
     Request request = requestAndRelatedRecords.getRequest();
 
     if (!request.allowedForItem()) {
-      return failed(failure(
-        String.format("%s requests are not allowed for %s item status combination", request.getRequestType().getValue() , request.getItem().getStatus().getValue()),
-        request.getRequestType().getValue(),
-        request.getItemId()
-      ));
+      //TODO: Investigate whether the parameters on this error are correct
+      return failedValidation(
+        format("%s requests are not allowed for %s item status combination",
+          request.getRequestType().getValue() , request.getItem().getStatus().getValue()),
```

Comment:
Likely it makes sense to define a variable to represent request.getRequestType().getValue() here